christfellowshipchurch/web-app-v2
# Quiz Plugin

### About
This PR demoes a sample quiz hosted on our website using the Typeform SDK. 

### Test Instructions
1. Go to `/quizzes/jJ7aDEPM-my-quiz` on the Vercel link
2. You should be able to view the sample quiz and also change the quiz title 

### Screenshots
<img width="1436" alt="Screenshot 2023-08-02 at 11 01 35 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/b7e5bf76-8e70-4eb8-afb6-f4cd0ec34332">


### Related Tickets
[CFDP-2652](https://christfellowshipchurch.atlassian.net/browse/CFDP-2652)
